config.in

```diff
@@ -1,2 +1,4 @@
 db sqlite3 /var/lib/soju/main.db
 message-store fs /var/lib/soju/logs/
+listen :6697
```

Comment:
Nit: I'd slightly prefer `listen ircs://` here instead, to avoid hardcoding a port number.